src/styles/header/header.scss

```diff
@@ -0,0 +1,63 @@
+.header {
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  display: flex;
+  box-sizing: border-box;
+}
+
+.nav {
+  display: flex;
+
+  &__item {
+    margin-right: 20px;
+
+    &:last-child {
+      margin-right: 0;
```

Comment:
Instead of removing the right margin from the last item, you can use the `gap` property to create distance between flex items. It's a more modern and flexible approach.